src/wallet/crypter.cpp

```diff
@@ -113,7 +113,7 @@ bool CCrypter::Decrypt(const std::vector<unsigned char>& vchCiphertext, CKeyingM
 }
 
 
-static bool EncryptSecret(const CKeyingMaterial& vMasterKey, const CKeyingMaterial &vchPlaintext, const uint256& nIV, std::vector<unsigned char> &vchCiphertext)
+bool CCryptoKeyStore::EncryptSecret(const CKeyingMaterial& vMasterKey, const CKeyingMaterial &vchPlaintext, const uint256& nIV, std::vector<unsigned char> &vchCiphertext)
```

Comment:
Sorry for not spotting this earlier. Modifying bitcoin code like that is not something we'd want to do. On the other hand adding another method to the `CCryptoKeyStore` is ok. This way you can even revert the change that makes `vMasterKey` protected instead of private

Reply:
Ok, will do that way. Thank you.